scripts/version-helper.js

```diff
@@ -6,11 +6,7 @@ const pkg = require('../package.json');
  * > manifest.version = `${pkg.version}.${pkg.beta}`
  */
 function getVersion() {
-  return `${pkg.version.match(/\d+\.\d+/)[0]}.${pkg.beta || 0}${
-  // Create a beta release with the same code as in stable release.
-  // Used in unlisted version.
-    process.env.BETA ? 'b' : ''
-  }`;
+  return `${pkg.version.match(/\d+\.\d+/)[0]}.${pkg.beta || 0}`;
```

Comment:
IIRC there is a problem here, if we release `2.14.0` as a listed version, we cannot release `2.14.0` also as a BETA version. That's why I added a suffix.

Unless we don't release the same code to BETA channel?

Reply:
I never released the same version in beta and AFAIK it's impossible now because releases intentionally end on `.0` while beta ends on a non-zero number. Even if the js code is the same between channels, manifest.json will be different anyway due to different version.

Reply:
Oh I already removed the additional BETA release in 8b332ce557a0313701e08396bc23a7e6e82af72a
Then no concern from my side.